### PR TITLE
Update to GA4 syntax

### PR DESCRIPTION
**Pull request by danajanssen**

### CODE REVIEW

File: app/views/layouts/_google_analytics.html.erb [hunk 1]
```diff
@@ -4,4 +4,5 @@
   window.dataLayer = window.dataLayer || [];
   function gtag(){dataLayer.push(arguments);}
   gtag('js', new Date());
+  gtag('config', "<%= code %>");
```
Comment (leom806) [inline app/views/layouts/_google_analytics.html.erb:7]: is this `code` still available? seems that it's now removed

Comment (danajanssen) [reply app/views/layouts/_google_analytics.html.erb:7]: it's in `effective_pages_helperl.rb#effective_pages_google_analytics`

File: app/assets/javascripts/effective_pages/google_analytics.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
   if(typeof(gtag) != 'function') { return }
 
-  var code = $('head').find('script[data-gtag-code]').first().data('gtag-code');
-  if(typeof(code) == undefined) { return }
-
-  gtag('config', code, {
-    'page_title': document.title,
-    'page_path': location.href.replace(location.origin, "")
+  gtag('event', 'page_view', {
+    page_title: event.target.title,
```
Comment (leom806) [inline app/assets/javascripts/effective_pages/google_analytics.js:7]: not sure if we want to `event.target?.title` on these attributes or not

Comment (matt-riemer) [reply app/assets/javascripts/effective_pages/google_analytics.js:7]: we always have page titles, should be okay, this is good feedback ty

**Pull request merged**